docs/guides/Graphql-Api-on-gRPC.md

```diff
@@ -0,0 +1,117 @@
+### Using gRPC with Tailcall
+
+Tailcall makes it easy to build GraphQL APIs that connect to gRPC services. With just a few lines of schema directives, you can call gRPC methods directly from your GraphQL resolvers.
+
+## Adding the gRPC Directive
+To call a gRPC method, use the @grpc directive on a GraphQL field:
+
+```type Query {
+  getUser(id: ID!): User @grpc(service: "UserService", method: "GetUser")
+}
+```
+This will call the GetUser method on the UserService gRPC service.
+
+You can specify the proto file path and gRPC server URL in your Tailcall config.
+
+## Mapping Parameters
+Tailcall will automatically map GraphQL arguments to gRPC request parameters.
+
+You can also manually map fields:
+
+```type Query {
+  getUser(id: ID!): User @grpc(
+    service: "UserService" 
+    method: "GetUser"
+    args: [{graphql: "id", grpc: "userId"}]
+  )
+}
+```
+
+## Handling Responses
+The gRPC response is mapped to GraphQL based on field names.
+
+Use @map directives to customize:
```

Comment:
Where did u read about the @map operator?

Reply:
I wrote it because as per the docs in /docs/operators/grpc

 By default,  the mapping of gRPC responses to GraphQL based on field names is done automatically by Tailcall and is not something you configure directly in the schema.

The docs are explaining that if your gRPC method returns a message type like:

message UserResponse {
  string id = 1;
  string name = 2; 
}
Then Tailcall will automatically map that to a GraphQL type like:

type User {
  id: ID
  name: String
}
Without you needing to specify any additional mapping.

The mapping happens under the hood when Tailcall processes the gRPC response. So there isn't a specific operation for it in the schema file itself.   

Or maybe I misread the info if its not available.